src/contracts/interfaces/IEigenPod.sol

```diff
@@ -4,6 +4,8 @@ pragma solidity >=0.5.0;
 import "../libraries/BeaconChainProofs.sol";
 import "./IEigenPodManager.sol";
 import "./IBeaconChainOracle.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
could we move events to interfaces for Pod stuff and delegation manager? I prefer it.

Wdyt @ChaoticWalrus

Reply:
yes please. I think this is "more correct" / better practice.

Reply:
will make a separate PR for this as well as other stuff mentioned: https://github.com/Layr-Labs/eignlayr-contracts/issues/962